mlflow/protos/databricks_fs_service.proto

```diff
@@ -0,0 +1,324 @@
+// Check Databricks Protobuf Style Guide (go/protostyleguide) about recommended Protobuf practices.
```

Comment:
Let's rename this to `databricks_filesystem_service.proto`

`fs` is overloaded between filesystem and feature store.